site/src/documents/pages/Page.tsx

```diff
@@ -33,11 +33,7 @@ export function Page(props: PropsWithLayout<GQLPageQuery>): JSX.Element {
                     canonicalUrl={`${process.env.NEXT_PUBLIC_SITE_URL}${props.pageContent?.path}`}
                 />
             )}
-            {document && document.__typename === "Page" ? (
-                <>
-                    <div>{document.content && <PageContentBlock data={document.content} />}</div>
-                </>
-            ) : null}
+            {document && document.__typename === "Page" ? <main>{document.content && <PageContentBlock data={document.content} />}</main> : null}
```

Comment:
```suggestion
            {document && document.__typename === "Page" ? <main>{<PageContentBlock data={document.content} />}</main> : null}
```

I believe `document.content` can never be undefined/null

Reply:
True 👍🏼